### PR TITLE
Enable SDK usage in browsers

### PR DESCRIPTION
**Pull request by Ivo-Yankov**
**Description**:
Adds a new Envoy proxy container, which redirects http calls from `http://127.0.0.1:50213` to the conesnsus address: `127.0.0.1:50211`.

**Related issue(s)**:

Fixes #192

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: test/cypress/src/client.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    Client,
+    AccountId,
+    PrivateKey,
+    Hbar,
+    AccountCreateTransaction,
+} from "@hashgraph/sdk";
+
+const operatorId = AccountId.fromString('0.0.2');
+const operatorKey = PrivateKey.fromString('302e020100300506032b65700422042091132178e72057a1d7528025956fe39b0b847f200ab59b2fdd367017f3087137');
+const client = Client.forNetwork({"http://127.0.0.1:50213":"0.0.3"}).setOperator(operatorId, operatorKey);
```
Comment (georgi-l95) [inline test/cypress/src/client.js:11]: nit: we can maybe get those from .env

**Review by georgi-l95 — APPROVED** [review]
Nice progress, looks good

File: test/cypress/src/client.js (hunk 1 above)
Comment (Ivo-Yankov) [reply test/cypress/src/client.js:11]: I thought about that, but the AccountId and private key should always be the same, probably the proxy address could be subject to change when we implement multiple consensus nodes. I suggest we leave it as it is for now.

Comment (Ivo-Yankov) [reply test/cypress/src/client.js:11]: I thought about that, but the AccountId and private key don't need to be configurable. The proxy address may be subject to change when we implment multiple consensus nodes. I think we can leave it as is for now.

**Pull request merged**